fastparquet/writer.py

```diff
@@ -737,11 +788,19 @@ def write(filename, data, row_group_offsets=50000000,
         resolution; in "int96" mode, they are written as 12-byte blocks, with
         the first 8 bytes as ns within the day, the next 4 bytes the julian day.
         'int96' mode is included only for compatibility.
+    quantization_level: int or None
+        If not None, then quantization_level floating point values in this many units of the
+        median absolute deviation of the column. Only works for the 'simple' file
+        scheme.
 
     Examples
     --------
     >>> fastparquet.write('myfile.parquet', df)  # doctest: +SKIP
     """
+
+    global COLUMN_MADS
+    COLUMN_MADS = {}
```

Comment:
Ahh woops! That is answer for another question. :(

I think defining the bin width relative to the standard deviation or MAD of the data is the right way to go. This leaves approximately the same fractional precision in each column regardless of the overall order of magnitude of each column. 

Absolute scaling is supported by FITS fpack routines which first used this technique, but I don't personally think it warrants inclusion here. Just my opinion though. I am sure there is a use case for it too. :)